tests/testsuite.src/used_binaries.at

```diff
@@ -1000,5 +1000,14 @@ AT_CHECK([$COBC -fdiagnostics-plain-output -fdiagnostics-show-caret -Wno-others
  > <EOF>
 ]])
 
+
+AT_CHECK([$COBC -fdiagnostics-plain-output -fdiagnostics-absolute-paths -Wall prog.cob 2> compiler.output], [1])
```

Comment:
`AT_CHECK([$COMPILE -fdiagnostics-absolute-paths prog.cob 2> compiler.output], [1])`

should work fine, no?

Reply:
ok